.github/workflows/create-release-branch.yaml

```diff
@@ -3,9 +3,11 @@ name: Confirm Snap Builds
 on:
   workflow_dispatch:
     inputs:
-      branch:
+      branches:
         type: string
-        description: Run on which branch
+        required: false
+        default: ""  # defaults to all matching branches (main and release branches)
+        description: Run on which k8s-snap branches (space separated)
```

Comment:
```suggestion
        description: Run on which k8s-snap branches (space separated). If empty, it will run on all matching branches (main and release branches).
```